lib/src/runner/vm/platform.dart

```diff
@@ -29,7 +30,12 @@ class VMPlatform extends PlatformPlugin {
   /// The test runner configuration.
   final _config = Configuration.current;
 
-  VMPlatform();
+  /// A function that returns a [PackageResolver] that is used in spawning
+  /// isolates based on the current test suite path.
+  final PackageResolver Function(String path) _packageResolver;
+
+  VMPlatform({PackageResolver Function(String path) packageResolver})
```

Comment:
The name here implies that it affects anything using a package resolver, but in fact it only affects the data isolate code path (one of 3 possible code paths).

It should probably be named accordingly?